src/types/venues.ts

```diff
@@ -178,6 +178,14 @@ export interface JazzbarVenue extends Venue {
   host: {
     icon: string;
   };
+  owners: string[];
+  theme?: {
+    primaryColor: string;
+    backgroundColor?: string;
+  };
+  id: string;
+  rooms?: AnyRoom[];
+  mapBackgroundImageUrl?: string;
```

Comment:
These seem unrelated to this PR/any changes I've seen in it? Where did they come from?

Reply:
I think this is a rebase/merge thing. I will double check and remove it if it's not being used.